staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/dispatcher.go

```diff
@@ -97,6 +97,13 @@ func (a *mutatingDispatcher) callAttrMutatingHook(ctx context.Context, h *v1beta
 		return &webhookerrors.ErrCallingWebhook{WebhookName: h.Name, Reason: fmt.Errorf("Webhook response was absent")}
 	}
 
+	for k, v := range response.Response.Annotations {
+		if err := attr.AddAnnotation(h.Name+"/"+k, v); err != nil {
```

Comment:
if the key already is already namespaced to the webhook, noop?

Reply:
erroring on an existing key means that a mutating and validating webhook with the same name cannot set the same key, since they share a namespace

Reply:
> erroring on an existing key means that a mutating and validating webhook with the same name cannot set the same key, since they share a namespace

Local mutating and validating admission may have same problem.
What about we trust the cluster admin here?